README.md

```diff
@@ -85,18 +85,68 @@ For example
 npx i18n-translate-generator translate "Hello world" "en" "greetings" -s "example/custom-setting.config.json"
 ```
 
+### Nested Translations
+
+You can nest translations by using the `.` character on the property name parameter, for example:
+
+```sh
+npx i18n-translate-generator translate "Account settings" "en" "accountSettings.title"
+```
+
+```json
+
+{
+  "accountSettings": {
+    "title": "Account settings"
+  }
+}
+```
+
+You can continue nesting translations as much as you want.
```

Comment:
One more nagging question I have  is what happens if I put the same key (one already used), for example, I try email again, will it silently substitute the key or gives some warning?

Reply:
By the moment silently substitute. But would be a great feature add any warning if that happens. Will check what could I do

Reply:
https://github.com/victor-heliomar/i18n-translation-generator/issues/11 Opened new issue to investigate this case!